app/controllers/help_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HelpController < ApplicationController
+  before_action :users_only, only: [:first_login]
+  layout false
+
+  def first_login
+  end
+
+  def preferences_locale
+  end
```

Comment:
This page isn't something we've shared with users yet, but if we're going to rename the others `preferences_topic` instead of `topic-preferences`, each time we do one, we should ask Systems to set up a redirect to ensure we don't end up with any broken links in places like old news posts or FAQs.